README.md

```diff
@@ -1 +1,42 @@
-# jv-dao-service
\ No newline at end of file
+# HW 03
+- Establish connection to Database.
+- Create class Client. Records should be stored in your Database.
+
+```java
+public class Client { 
+    private Long id;
+    private String name;
+    private BigDecimal amount;
+    private String accountNumber;
+}
+```
+- Create `BankSystemDao` and `BankSystemService`, and their implementations.
+
+DAO methods
+```java
+- Optional<Client> register(Client client);
+- Optional<Client> find(Long id);
+- Optional<Client> findByAccount(String accountNumbet);
+- List<Client> getAll();
```

Comment:
It doesn't look good. The student can retrieve all users in runtime and sort on the server-side. Maybe it's better to remove it?